.travis.yml

```diff
@@ -5,19 +5,14 @@ rust:
   - stable
   - beta
   - nightly
-env:
-  - CARGO_DIR=rust/native
-  - CARGO_DIR=c/rust
 install:
   - rustup component add rustfmt clippy
-before_script:
-  - cd $CARGO_DIR
+  - eval $(gimme stable)
```

Comment:
Give you a stable? Nay! :D

Seriously though. If I understand correctly, this sets up the environment so that the user will have access to a stable Go compiler? That makes sense, but it sure would be nice to have a comment explaining it...